lang/ar.js

```diff
@@ -0,0 +1,25 @@
+/* eslint quotes: 0 */
+
+export default {
+	"components.alert.alert.close": "إغلاق التنبيه",
```

Comment:
The `components` part of this NS is kind of ambiguous.  `core` seems more meaningful, but I know this matches the folder structure, and also that not everything is a `component`.

Reply:
Haha I asked Margaree to put `components` in as part of the original draft review. Because we're going to have page templates, mixins and other stuff at the top level. `core` is redundant since this lang file is only used within core.